### ISSUE TITLE
(Bug) Countdown shows is  incorrect if time period more than 30 days

### ISSUE THREAD

**Issue by dennis00010011b** [poanetwork/token-wizard#764]
####Steps for reproduce:
1. Create  crowdsale with start time more than 30 days in future
2. Open invest page and observe countdown timer
Expected result: 
  - countdown timer should show time period before start of crowdsale
Actual result: 
 - countdown timer shows 29 days 23 hours 59 minutes

If you are reporting a problem with Token Wizard, please include the following information:

### Which network did you use? (Mainnet, Kovan, Rinkeby, etc.)

*Your answer*

### 
https://wizard.oracles.org/invest?addr=0xDE6a49A44A866D3EC050a7fbAD4c08E786026eF7&networkID=77

*Your answer*

<img width="1440" alt="screen shot 2018-04-03 at 09 46 56" src="https://user-images.githubusercontent.com/14067096/38263789-ddedb632-3725-11e8-93f9-2ff1a4f893cd.png">
<img width="1440" alt="screen shot 2018-04-03 at 09 46 41" src="https://user-images.githubusercontent.com/14067096/38263790-e0a0c1a8-3725-11e8-991c-7db0d42ebaa6.png">

*Your answer*

### Do you see errors in the dev console? If yes, please include a screenshot

No

If you see errors, please right click on them and "Save as..". Zip saved file and attach it to the Issue.

No

---

**Comment by fvictorio** [comment]
Makes sense. The problem must be that we are using `.days` instead of `.asDays` ([see here](https://momentjs.com/docs/#/durations/days/)). Should be an easy fix.
